src/master-playlist-controller.js

```diff
@@ -370,7 +336,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     // Pause any alternative audio
     if (this.audioPlaylistLoader_) {
       this.audioPlaylistLoader_.pause();
-      this.audioPlaylistLoader_ = null;
+      this.audioPlaylistLoader_.dispose();
```

Comment:
The problem with using dispose here is that we reuse the alternate audio playlist loader the if it is selected in the future (we don't create them when they are selected).

It would also still need to be set to null for some other checks to work (it isn't the optimal approach, but right now we use if (this.audioPlaylistLoader_) to determine if we are using an alternate audio playlist.